src/test/java/pro/trevor/tankgame/util/TankBuilder.java

```diff
@@ -0,0 +1,46 @@
+package pro.trevor.tankgame.util;
+
+import org.json.JSONObject;
+
+import pro.trevor.tankgame.rule.impl.version3.Tank;
+import pro.trevor.tankgame.rule.impl.version3.TankAttribute;
+import pro.trevor.tankgame.state.board.Position;
+import pro.trevor.tankgame.state.board.attribute.IAttribute;
+import pro.trevor.tankgame.state.board.unit.GenericTank;
+
+public class TankBuilder<T extends GenericTank<E>, E extends Enum<E> & IAttribute> 
+{
```

Comment:
open braces should be on the same line as the function or class declaration

Reply:
ACK